ICRC-1.did

```diff
@@ -4,12 +4,16 @@ type Timestamp = nat64;
 // Number of nanoseconds between two [Timestamp]service
 type Duration = nat64;
 
-type SubAccount = blob;
+type Subaccount = blob;
+
+type Account = record {
+    principal: principal;
```

Comment:
The candid file doesn't pass validation. I think we need to put the field name into quotes because it's a reserved symbol.
```suggestion
    "principal": principal;
```

Reply:
Ah, I didn't even think of running it through didc. I've updated the type to have quotes and ran didc locally and it all passed. This should be good now!